jaxrs/src/main/java/com/cerner/beadledom/jaxrs/exceptionmapper/WebApplicationExceptionMapper.java

```diff
@@ -0,0 +1,63 @@
+package com.cerner.beadledom.jaxrs.exceptionmapper;
+
+import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+
+import com.cerner.beadledom.json.common.models.JsonError;
```

Comment:
Seems like maybe I didn't completely fix checkstyle; the updated Google styleguide only uses two groups (static and normal) with each group ordered alphabetically.

I believe this gist contains an Intellij formatter with the new import order: https://gist.github.com/johnlcox/b7ec8840c74c43df286cbfc52225ed09

Reply:
I'm assuming this is what you mean:
```
package com.cerner.beadledom.jaxrs.exceptionmapping;

import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;

import com.cerner.beadledom.json.common.model.JsonError;
import javax.ws.rs.WebApplicationException;
import javax.ws.rs.core.MediaType;
import javax.ws.rs.core.Response;
import javax.ws.rs.ext.ExceptionMapper;
import javax.ws.rs.ext.Provider;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
```

Is this the style that every class should follow moving forward, including Scala classes? Or just Java?

Reply:
Grouped imports using the provided GoogleStyle formatter: https://github.com/cerner/beadledom/pull/45/commits/6a5509f6056635e81122e800492dfdd530d2dedf